src/plugins/utilities/layout/centered-container.js

```diff
@@ -19,6 +19,8 @@ module.exports = function () {
     const newUtilities = {
       // Center an element horizontally.
       '.centered-container, .cc': {
+        paddingLeft:  theme('decanter.screenMargins.xs'),
+        paddingRight:  theme('decanter.screenMargins.xs'),
```

Comment:
To work with removing "xs" from screens. I took the xs value out of the for loop for media query. xs doesn't need to be wrapped in a media query since we're doing mobile first.